data_adapter_oemof/build_datapackage.py

```diff
@@ -289,6 +313,11 @@ def save_datapackage_to_csv(self, destination: str) -> None:
                 index=True,
                 sep=";",
             )
+        if "timeindex" in self.tsa_parameters:
```

Comment:
`self.tsa_parameters` is None and therefore not iterable. Can you check this?

Reply:
Thank you for pointing out! :)